go/pkg/pwdb/db.go

```diff
@@ -1,41 +1,60 @@
 package pwdb
 
 import (
+	"log"
+	"reflect"
+	"strings"
+
 	"github.com/bwmarrin/snowflake"
-	"github.com/jinzhu/gorm"
-	"go.uber.org/zap"
-	"moul.io/zapgorm"
+	"gorm.io/gorm"
+	"gorm.io/gorm/schema"
 	"pathwar.land/pathwar/v2/go/pkg/errcode"
 )
 
-type Opts struct {
-	Logger *zap.Logger
-
-	// internal
-	skipFK bool
+// DefaultGormConfig that fits on both prod and test
+var DefaultGormConfig = gorm.Config{
```

Comment:
`DefaultSomething` is a good name for a const that is used to get the default values, but since it's changed by another code it should not be named `DefaultSomething`

+ make everything you can to not use a global var